stix_shifter_modules/trellix_endpoint_security_hx/stix_translation/json/operators.json

```diff
@@ -0,0 +1,13 @@
+{
+    "ComparisonExpressionOperators.And": "and",
+    "ComparisonExpressionOperators.Or": "or",
+    "ComparisonComparators.GreaterThan": "greater than",
+    "ComparisonComparators.LessThan": "less than",
+    "ComparisonComparators.Equal": "equals",
+    "ComparisonComparators.NotEqual": "not equals",
+    "ComparisonComparators.Like": "contains",
+    "ComparisonComparators.In": "equals",
+    "ComparisonComparators.Matches": "contains",
+    "ObservationOperators.Or": "or",
+    "ObservationOperators.And": "or"
```

Comment:
This seems incorrect to me. 

> "ObservationOperators.And": "or"

This seems to be a contradiction setting an AND operator as an OR operator

Reply:
It is as designed. AND operator between two observation statements is converted to OR. All the connectors are following the same.